apps/t_shirt_delivery/templates/registration/change_tshirt.html

```diff
@@ -103,21 +103,21 @@
 
             // Listen for button click
             newButton.addEventListener('click', () => {
-                selected_year = newButton.innerText
+                selectedYear = newButton.innerText
                 // Remove previous active button CSS
-                if (current_button) {
-                    current_button.classList.remove('btn-primary')
+                if (currentButton) {
+                    currentButton.classList.remove('btn-primary')
                 }
                 // Save current button  as last clicked and add style
-                current_button = newButton
-                current_button.classList.add('btn-primary')
+                currentButton = newButton
+                currentButton.classList.add('btn-primary')
 
                 // Delete all Tshirt SELECT option
                 removeAll()
 
                 // Set new SELECT OPTIONS based on the year
-                for (let i=0; i < years_and_options[selected_year].length; i++) {
-                    add(years_and_options[selected_year][i], years_and_options[selected_year][i])
+                for (let i=0; i < yearAndOptions[selectedYear].length; i++) {
+                    add(yearAndOptions[selectedYear][i], yearAndOptions[selectedYear][i])
```

Comment:
`add(option_object)` function has only one parameter `option_object`  in function declaration, but you call this function here with two args.

`add(yearAndOptions[selectedYear][i], yearAndOptions[selectedYear][i])`

should be  

`add(yearAndOptions[selectedYear][i])`